run.yml

```diff
@@ -140,3 +140,7 @@
     ansible.builtin.shell:
       cmd: swapoff -a
 
```

Comment:
1.  Please remove the new line, as this is the file convention. 
2.  If this playbook is going to be merged upstream, probably this won't be allowed since other teams are forking it and may want to keep the iso for their own use.
Hence, I believe that it should be in our playbooks. (in the last part in a teardown role or similar)

Reply:
moved this task to the steal's playbook, therefor closing this PR.